db/test.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python2
```

Comment:
non-blocking for this PR, but at some point we might want to upgrade to python 3 🐍